core/kazoo_amqp/src/api/kapi_acdc_queue.erl

```diff
@@ -60,7 +60,7 @@
 
 -export([queue_size/2, shared_queue_name/2]).
 
--include("acdc.hrl").
+-include("../../applications/acdc/src/acdc.hrl").
```

Comment:
we have to figure out a way to decouple this. we can't have core referring an app